modules/cnf-troubleshooting-resource-topo-exporter.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: PROCEDURE
+[id="cnf-troubleshooting-resource-topo-exporter_{context}"]
+= Troubleshooting the resource topology exporter
+
+Troubleshoot `noderesourcetopologies` objects where unexpected results are occurring by inspecting the corresponding `resource-topology-exporter` logs.
+
+[NOTE]
+====
+NUMA resource topology exporter instances in the cluster should be named for nodes they refer to. For example, a worker node with the name `worker` should have a corresponding `noderesourcetopologies` object called `worker`.
```

Comment:
Avoid "should". Try "recommended" in the first sentence. I believe it's okay in the second sentence as an example.